### ISSUE TITLE
Can I get the checkpoint of your trained model?

### ISSUE THREAD

**Issue by wxx07** [mheinzinger/SeqVec#3]
Hi, 
I assume the weights([weights.hdf5] file) you released of the pre-trained model does not contain the 'Softmax' part (the part to output the next token when training)  if you dumped the weights as allennlp recommanded.
I would be really thankful if I could get the checkpoint of your pre-trained model.

Bruce

**Comment by sacdallago** [comment]
- embedder:
  - weights: http://maintenance.dallago.us/public/embeddings/embedding_models/seqvec/weights.hdf5
  - options: http://maintenance.dallago.us/public/embeddings/embedding_models/seqvec/options.json subcellular 
- location checkpoint: http://maintenance.dallago.us/public/embeddings/feature_models/seqvec/subcell_checkpoint.pt
- secondary structure checkpoint: http://maintenance.dallago.us/public/embeddings/feature_models/seqvec/secstruct_checkpoint.pt

We are working on a `pip install`able software package for you to easily have access to the whole architecture. It's just taking us a bit longer than originally planned. But coming...

**Comment by sacdallago** [comment]
Closing due to inactivity

**Comment by mheinzinger** [comment]
The checkpoint files are now available under:
https://github.com/allenai/allennlp/blob/master/allennlp/commands/elmo.py
